packages/appstore/src/components/add-app/stories/add-app.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import type { Meta, Story } from '@storybook/react';
+import AddApp from '../index';
+
+type AddAppProps = Parameters<typeof AddApp>[0];
+
+export default {
+    title: 'AddApp',
+    argTypes: {
+        is_dark_mode_on: {
+            description: 'Optional',
+            type: 'boolean',
+            defaultValue: false,
+        },
+        is_mobile: {
+            description: 'Optional',
+            type: 'boolean',
+            defaultValue: false,
+        },
+    },
+} as Meta<AddAppProps>;
+
+const Template: Story<AddAppProps> = args => <AddApp {...args} />;
+
+export const AddAppTemplate = Template.bind({});
+AddAppTemplate.args = {
+    apps: [
+        {
+            app_type_title: 'Deriv X Apps',
+            linked_apps: [
+                { app_name: 'Deriv MT5 Synthetics', app_icon: 'icDxtradeSynthetic' },
```

Comment:
can we change from `app_name` to `name`?  
We call `Deriv X` an app. So, `Deriv MT5 Synthetics` shouldn't be an app as well.

Reply:
@balakrishna-binary I named them after the types created in `AppCard` before, you mean I should rename those too?

Reply:
Oh okay. It's fine then. We can refactor it later